client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js

```diff
@@ -112,12 +112,12 @@ const UnverifiedAddress = ( {
 						) }
 					</p>
 					{ uspsUrl && (
-						<ExternalLink icon={ true } href={ uspsUrl } target="_blank">
-							{ translate( 'Verify with USPS' ) }
+						<ExternalLink href={ uspsUrl } target="_blank">
+							{ translate( 'Verify with USPS ' ) }
 						</ExternalLink>
 					) }
-					<ExternalLink icon={ true } href={ googleMapsUrl } target="_blank">
-						{ translate( 'View on Google Maps' ) }
+					<ExternalLink href={ googleMapsUrl } target="_blank">
+						{ translate( 'View on Google Maps ' ) }
```

Comment:
I'm not seeing much of a difference with/without the blank space added here and above - in which case does it become necessary? Would it affect the translation strings?
Without space: https://d.pr/i/4P4dNY
With space: https://d.pr/i/SNat4H

Reply:
This is a mistake. Thanks for noticing. Fixed.